apps/web/src/pages/SearchRequests/ViewSearchRequestPage/components/ViewSearchRequest.tsx

```diff
@@ -164,23 +164,22 @@ const ManagerInfo = ({
                 />
                 <FilterBlock
                   title={intl.formatMessage({
-                    defaultMessage: "Date done",
-                    id: "BAzKWq",
+                    defaultMessage: "Status last changed",
+                    id: "pAW6AU",
```

Comment:
Not entirely sure what to do here, should I drop this block altogether instead?

Reply:
I recommend checking with @Jerryescandon  but I suggest moving the date right beside to the text label of the status to show they're strongly related.  Something like
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/66a0b605-0d2a-46b2-b143-981315b9d4b1)

Reply:
Adjusted to fit the screenshot by Jerbo below in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7210/commits/b27f175514c9a8d3e17754f93a563092b0e42926

One caveat, I preserved the datetime view since we use it elsewhere and I thought it should stay consistent

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/2e3486bd-19bc-4315-b4f9-8ae2fbde7efb)

Reply:
Love it!